src/main/java/io/vertx/rabbitmq/impl/RabbitMQClientImpl.java

```diff
@@ -46,9 +40,46 @@ public RabbitMQClientImpl(Vertx vertx, JsonObject config) {
   }
 
   @Override
+  public boolean isConnected() {
+    boolean connected = false;
+    if (connection != null) {
+      if (connection.isOpen()) {
+        connected = true;
+      }
+    }
+    return connected;
+  }
+
+  @Override
+  public boolean isOpenChannel() {
+      return channel.isOpen();
+  }
+
+  @Override
+  public void basicAck(long deliveryTag, boolean multiple, Handler<AsyncResult<JsonObject>> resultHandler) {
+    forChannel(resultHandler, (channel) -> {
+      channel.basicAck(deliveryTag, multiple);
+      return null;
+    });
+  }
+
+  @Override
+  public void basicNack(long deliveryTag, boolean multiple, boolean requeue, Handler<AsyncResult<JsonObject>> resultHandler) {
+    forChannel(resultHandler, (channel) -> {
+      channel.basicNack(deliveryTag, multiple, requeue);
+      return null;
+    });
+  }
+
+  @Override
   public void basicConsume(String queue, String address, Handler<AsyncResult<Void>> resultHandler) {
+    basicConsume(queue, address, true, resultHandler);
```

Comment:
The docs you added in the interface specify that the basicConsume(...) method with 3 parameters produces an explicit acknowledgement consumer. Therefore the 3. parameter should be false here, shouldn't it?